ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -418,9 +334,10 @@ public struct Avatar: View {
         return Int(abs(javaHashCode(combinedHashable)))
     }
 
-    private static func initialsCalculatedColor(fromPrimaryText primaryText: String?, secondaryText: String?, colorOptions: [UIColor]? = nil) -> UIColor {
+    private static func initialsCalculatedColor(fromPrimaryText primaryText: String?, secondaryText: String?, colorOptions: [DynamicColor]? = nil) -> DynamicColor {
         guard let colors = colorOptions else {
-            return .black
+            // return black if there are no color options
+            return .init(light: ColorValue(0x000000))
```

Comment:
```swift
return .init(light: fluentTheme.globalTokens.neutralColors[.black])
```

This'll probably require making this function non-static so it can access `fluentTheme`, but that should be fine.

Reply:
Thanks for the suggestion! Fixing this also helped me notice I forgot to update the theme to a FluentTheme...